mysql-test/suite/rocksdb/t/lock_wait_timeout_stats.test

```diff
@@ -0,0 +1,27 @@
+--source include/have_rocksdb.inc
+
+#
+# MyRocks-specific tests for locking
+#
+--source include/have_debug.inc
+
+--enable_connect_log
+--disable_warnings
+drop table if exists t;
+--enable_warnings
+create table t (a int not null primary key) engine=rocksdb;
+
+begin;
+insert into t values (0);
+
+--connect (con1,localhost,root,,)
+--connection con1
+set @@rocksdb_lock_wait_timeout=1;
+begin;
+--error ER_LOCK_WAIT_TIMEOUT
+insert into t values(0);
+select INNODB_ROW_LOCK_WAIT_TIMEOUTS from information_schema.table_statistics where table_engine="ROCKSDB" and table_name="t";
+select INNODB_ROW_LOCK_WAIT_TIMEOUTS from information_schema.table_statistics where table_engine="ROCKSDB" and table_name="t";
+--disconnect con1
+--connection default
+drop table t;
```

Comment:
... --source include/wait_until_count_sessions.inc somewhere after the disconnect for the testcase stability